chrome/content/zotero/xpcom/style.js

```diff
@@ -399,6 +431,72 @@ Zotero.Styles = new function() {
 			}
 		});
 	}
+	
+	/**
+	 * Populate menulist with locales
+	 */
```

Comment:
Not sure you're answering the right question here. Seems like we're talking about https://github.com/zotero/zotero/pull/623/files#r29601259